endpoint_product_catalog/views/ir_filters.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="product_assortment_view_form" model="ir.ui.view">
+        <field name="model">ir.filters</field>
+        <field
+            name="inherit_id"
+            ref="product_assortment.product_assortment_view_form"
+        />
+        <field name="arch" type="xml">
+            <div name="buttons" position="inside">
+                <button
+                    class="oe_stat_button"
+                    name="button_display_endpoints"
+                    icon="fa-star"
+                    type="object"
+                    attrs="{'invisible': [('endpoint_count', '=', 0)]}"
```

Comment:
this should be invisible if this is not an assortment filter

Reply:
Well, `endpoint_count` is always 0 for non-assortment filters, so it's implicit. But yeah, I'll make it explicit.